npm_and_yarn/spec/dependabot/npm_and_yarn/file_fetcher_spec.rb

```diff
@@ -79,21 +79,16 @@
       described_class.new(source: source, credentials: credentials, repo_contents_path: repo_contents_path)
     end
 
+    before do
+      allow(Dependabot::SharedHelpers).to receive(:run_shell_command).and_call_original
+    end
+
     it "pulls files from lfs after cloning" do
       # Calling #files triggers the clone
       expect(file_fetcher_instance.files.map(&:name)).to contain_exactly("package.json", "yarn.lock", ".yarnrc.yml")
-      expect(
-        File.read(
-          File.join(repo_contents_path, ".yarn", "releases", "yarn-3.2.4.cjs")
-        )
-      ).to start_with("#!/usr/bin/env node")
-
-      # LFS files not needed by dependabot are not pulled
```

Comment:
I'd kinda hate to lose this test, it's nice to ensure we are only getting what we want if someone refactors that call to `git lfs` in some way

Reply:
If someone refactored the `git lfs` call and forgot to use `--include` with explicit arguments somehow, they'd break the new test too. But yeah, a realworld test always covers you better. I'm fine with anything that puts the spec back to green.